setup/ReShade Setup.csproj

```diff
@@ -37,6 +39,12 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.WindowsAPICodePack, Version=1.1.2.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Ah I see. Indeed a problematic situation. MS released the code for that on their (now defunct) code.msdn.microsoft.com site, and someone copied it to github. See: https://github.com/aybe/Windows-API-Code-Pack-1.1 There are other repos with the same code. It could help with adding the dialog as source to the project so no external dependency is needed :)